packages/ui/core-components/src/lib/atoms/inputs/date-range/DateRange.svelte

```diff
@@ -8,12 +8,12 @@
 	import { INPUTS_CONTEXT_KEY } from '@evidence-dev/component-utilities/globalContexts';
 	import { QueryStore } from '@evidence-dev/query-store';
 	import { getQueryFunction } from '@evidence-dev/component-utilities/buildQuery';
-	import { getLocalTimeZone } from '@internationalized/date';
 	import HiddenInPrint from '../shared/HiddenInPrint.svelte';
 	import { page } from '$app/stores';
 
+	/** @param {Date} date */
 	function dateToYYYYMMDD(date) {
-		return date.toISOString().split('T')[0];
+		return `${date.getFullYear()}-${('0' + (date.getMonth() + 1)).slice(-2)}-${('0' + date.getDate()).slice(-2)}`;
```

Comment:
This template string is difficult to read, it would benefit from being broken up

Reply:
Done